.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Run Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    env:
+      GOFLAGS: "-mod=vendor"
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+    # if we don't do this, `gofmt` will want to rewrite all Go files due to bad line endings,
+    # because Git will convert all line endings to CRLF when cloning on windows
+    - name: Set Git to use Linux-style line endings
+      run: |
+        git config --global core.autocrlf false
+        git config --global core.eol lf
+    - name: Install Go
+      uses: actions/setup-go@v2-beta
```

Comment:
@bkabrda maybe we need to change the `GOPATH` to a place where the website can be cloned.